cmd/src/debug.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+)
+
+var debugCommands commander
+
+func init() {
+	usage := `'src debug' gathers and bundles debug data from a Sourcegraph deployment for troubleshooting.
+
+Usage:
+
+	src debug command [command options]
+
+The commands are:
+
+	kube                 generates kubectl outputs
+	comp                 generates docker outputs
+	serv                 generates docker outputs
```

Comment:
Similarly would call this `single-server`

Reply:
Compromising on this one for server, we usually refer to these as single-container or server in my experience, open to being convinced here though